src/components/VmDialog/index.js

```diff
@@ -138,6 +157,14 @@ class VmDialog extends React.Component {
     return actionUniqueId
   }
 
+  setUiError (name, error) {
+    this.setState({
+      uiErrors: Object.assign({}, this.state.uiErrors, {
```

Comment:
If you work with previous state, please, use function style for setState. 
https://medium.com/@wisecobbler/using-a-function-in-setstate-instead-of-an-object-1f5cfd6e55d1